packages/core/stories/layout/flex-layout.stories.tsx

```diff
@@ -94,3 +107,37 @@ FlexLayoutSimpleUsage.args = {
   gap: 3,
   wrap: true,
 };
+
+export const ContactDetailsExample = ({ index }: { index: number }) => (
+  <ContactDetails embedded={true}>
+    <ContactAvatar />
+    <ContactPrimaryInfo text={`Contact ${index + 1}`} />
+    <ContactSecondaryInfo text="Blackrock Advisors (UK) Limited" />
+    <ContactTertiaryInfo text="SPN 2188538" />
+    <ContactMetadata collapsible={true}>
+      <ContactMetadataItem value="Analyst" label="Role" />
+      <ContactMetadataItem value="London, GBR" label="Location" />
+      <ContactMetadataItem value="+44 2077 431102" label="Office" />
+
+      <ContactMetadataItem value="alex.brailescu@blackrock.com" label="Email" />
+    </ContactMetadata>
+  </ContactDetails>
+);
+
+const ContactCards: ComponentStory<typeof FlexLayout> = (args) => {
+  return (
+    <FlexLayout {...args}>
+      {Array.from({ length: 12 }, (_, index) => (
+        <FlexItem grow={1} key={index}>
```

Comment:
I think in this case we probably want the "grow" behaviour so the cards take up the whole space, otherwise it looks a bit odd i.e.
<img width="1778" alt="Screenshot 2022-06-08 at 14 44 00" src="https://user-images.githubusercontent.com/31695437/172632208-15cd183e-2205-4fe0-aab2-ee76c1ee68e2.png">